src/onDraw/dataExport/exportToXLSX/defineReportXLSX/createWS.js

```diff
@@ -23,10 +23,12 @@ export default function createWS(id) {
     const stylesheet = crfHeatMap().style.textContent;
     this.export.data.forEach((d, row) => {
         this.export.cols.forEach((variable, col) => {
-            const value = d[variable];
+            const value_col = value_cols.indexOf(variable) > -1 ? true : false;
+            const value =
+                value_col && !isNaN(d[variable + '_value']) ? d[variable + '_value'] : d[variable];
```

Comment:
which variables are being kicked out here?

Reply:
getting rid of the cols that aren't crf or query information since only those are numeric and colored etc